src/utils/update/fetch.beneficial.owners.private.data.ts

```diff
@@ -26,6 +26,7 @@ export const fetchBeneficialOwnersPrivateData = async (appData: ApplicationData,
 
 const mapBeneficialOwnersPrivateData = (boPrivateData, appData: ApplicationData) => {
   const mapIndividualOwners = beneficialOwner => {
+    logger.info(`************* beneficialOwner.ch_reference: ${beneficialOwner.ch_reference} ***************`);
```

Comment:
For consistency, suggest adding a similar line in the Gov/Corporate function, around line 35.